src/module.js

```diff
@@ -70,40 +71,43 @@ const createCssRules = (wizardConfig, extractSass) => ({
   })
 });
 
-const createScssRules = (wizardConfig, extractSass) => ({
-  test: /\.scss$/,
-  use: extractSass({
-    fallback: 'style-loader',
-    use: [
-      {
-        loader: 'css-loader',
-        options: {
-          camelCase: true,
-          localIdentName: '[local]-[hash:base64:5]',
-          minimize: wizardConfig.isProd,
-          modules: true
-        }
-      },
-      {
-        loader: 'postcss-loader',
-        options: postcssOptions
-      },
-      {
-        loader: 'resolve-url-loader'
-      },
-      {
-        loader: 'sass-loader',
-        options: {
-          data: `@import '${wizardConfig.input.stylesGlobals}';`,
-          includePaths: [
-            wizardConfig.input.styles
-          ],
-          sourceMap: false
+const createScssRules = (wizardConfig, extractSass) => {
+  const globalStylesPath = resolveFilePathIfExists(wizardConfig.input.stylesGlobals);
```

Comment:
I'm wondering whether it would have been better to create this file if it does not exist. But maybe not. What do you think?

Reply:
I was pretty astonished at globals.scss is being required, that's why I've introduced this check. In some projects you do not need global variables, so what is the purpose to require this file?